tests/crosstest/kitex2dubbo/container_type_test.go

```diff
@@ -208,22 +208,22 @@ func TestEchoBool2BoolMap_Java(t *testing.T) {
 }
 
 // dubbo-java -> dubbo-java does not support
-//func TestEchoBool2ByteMap_Java(t *testing.T) {
-//	req := map[bool]int8{
-//		true: 1,
-//	}
-//	resp, err := cli2Java.EchoBool2ByteMap(context.Background(), req)
-//	assertEcho(t, err, req, resp)
-//}
+func TestEchoBool2ByteMap_Java(t *testing.T) {
```

Comment:
```TestEchoBool2ByteMap_Java``` and ```TestEchoBool2Int16Map_Java``` should be commented since dubbo-java -> dubbo-java does not support?

Reply:
okok